test/integration/controller/job/job_controller_test.go

```diff
@@ -464,100 +461,6 @@ var _ = ginkgo.Describe("Job controller for workloads with no queue set", func()
 			return k8sClient.Get(ctx, wlLookupKey, createdWorkload)
 		}, util.Timeout, util.Interval).Should(gomega.Succeed())
 	})
-	ginkgo.When("The parent-workload annotation is used", func() {
```

Comment:
@alculquicondor We can not run integration test cases related to parent-workload annotation since we must create a known workload owner (currently, only mpijobs) as a parent job.